src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super();
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
also, you can do the method hidden reusable if you make it to change the hidden value to opposite(optional)